uktrade/data-hub-frontend
# Mandatory notes field in the "Send a referral" form.

## Description of change
The notes text area field should now be a mandatory field in the "Send a referral" form.

## Test instructions
Go to send a referral by:
1. Click a company
2. Remove the `/activity` at the end of the url
3. Add `/send-referral` at the end of the url
4. Try to submit with no notes added - you should see the error
5. Add notes and remove them - you should see the error

_What should I see?_

## Screenshots
![Screenshot 2020-03-17 at 16 50 33](https://user-images.githubusercontent.com/10154302/76880470-9e9d7800-686f-11ea-802d-4733d09614fb.png)

## Checklist

[//]: # "When submitting a PR make sure the code review guidelines have been satisfied.
https://github.com/uktrade/data-hub-frontend/blob/master/docs/Code%20review%20guidelines.md"
